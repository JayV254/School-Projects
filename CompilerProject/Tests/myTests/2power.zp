int main()
{
     puts("Find powers of 2 easily");
     puts("\n");
     puts("Enter an integer to be the exponent");

     int counter;
     int max_count;
     int result;
     result = 1;

     puts("\n");
     max_count = getnum();

     for(counter = 0; counter < max_count; counter = counter + 1)
     {
        result = result * 2;
     }
     putn(result);
     return 0;
}



