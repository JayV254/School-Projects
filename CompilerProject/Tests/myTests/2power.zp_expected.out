Find powers of 2 easily
Enter an integer to be the exponent
input-> 10
  1024 
